docs/aws-region-enable.md

```diff
@@ -52,6 +52,16 @@ In case of RHCOS:
 - Account: 531415883065
   ARN: arn:aws:iam::531415883065:user/art-rhcos-ci-s3-bot
   Refer 'RH-DEV AWS Account' in Bitwarden for User ID
+- Note:
+   - Make sure to add the new regions link in the Tenant Egress of the PROD Cluster to avoid any TCP timeout.
+   - Also, update the Tenant Egress directory with the details of the new aws regions and the DPP ticket.
```

Comment:
What is the difference between this step and the bullet point above?

Reply:
The first point is to update the tenantegress file in the cluster. The second is to update the Tenant egress directory in the internal repo. If it is not clear, I can rephrase it better.

Reply:
We should enforce the practice of only updating the TenantEgress file in the internal repo. We can then deploy the template to enact the change on the cluster after it's been reviewed in the repo first. 
So maybe this could be worded as something like "Follow the internal process to add the new region endpoint URL to the TenantEgress of the Prod Cluster."

Reply:
I have rephrased it. Hope its worded better.